adminSiteClient/gdocsDeploy.ts

```diff
@@ -40,21 +55,19 @@ export const checkIsLightningUpdate = (
         linkedDocuments: true,
         relatedCharts: true,
         revisionId: true,
-        // "tags" is not actually a lightning prop, as it requires rebaking other parts of the site,
-        // but they're set via the /setTags route and so should be ignored here
-        tags: true,
         updatedAt: true,
-        createdAt: false,
+        createdAt: false, // weird case - can't be updated
         id: false, // weird case - can't be updated
+        tags: false, // could require updating datapages
```

Comment:
I think I made the wrong call on the theoretical problem with tags before. If a document's tags are good in the preview, and I hit publish, that's what I would expect to be published. (As opposed to the tags that could have theoretically been set in the meantime)

So now the tags from the published JSON are what get persisted.

This still doesn't actually work with lightning updates because as far as the preview's concerned, the before and after gdocs will always have the same tags. But that's a edge case bug to fix later I think.